pages/linux/apache2ctl.md

```diff
@@ -0,0 +1,24 @@
+# apache2ctl
+
+> The CLI tool to administrate HTTP web server Apache.
+> This commmand comes with Debian based OS, for RHEL based OS see httpd.
```

Comment:
This line can be improved slightly. @sbrl / @pxgamer - Any suggestion ?

Also, what do you mean by "RHEL based OS" ? I thought RHEL was just an OS ? Are there variants of RHEL ?

Reply:
RHEL is an OS. But it looks like there are some others derived from it's code.
https://en.m.wikipedia.org/wiki/Red_Hat_Enterprise_Linux_derivatives

As for the description, I'm not sure. :thinking:

Reply:
Yes @pxgamer @agnivade  there are lot of other distros which are derived from Redhat's RHEL distro like CentOS, Fedora, Scientific Linux, Amazon Linux etc. These distros use **yum** for package management. Other popular distros are based on Debian like Ubuntu, Linux Mint, Elementory OS etc which use **apt-get** for package management.

So my above written line makes sense from this point of view.